can/interfaces/usb2can/usb2canInterface.py

```diff
@@ -118,10 +117,10 @@ def __init__(
         self.channel_info = f"USB2CAN device {device_id}"
 
         connector = f"{device_id}; {baudrate}"
-        self.handle = self.can.open(connector, flags_t)
+        self.handle = self.can.open(connector, flags)
 
         super().__init__(
-            channel=channel, dll=dll, flags_t=flags_t, bitrate=bitrate, *args, **kwargs
+            channel=channel, dll=dll, flags=flags, bitrate=bitrate, *args, **kwargs
```

Comment:
Since `BusABC` does not know these, we can also ignore them here.

```suggestion
            channel=channel, bitrate=bitrate, *args, **kwargs
```

Reply:
I've also removed bitrate as it looks like `BusABC` doesn't use this either, so now this line is `super().__init__(channel=channel, **kwargs)`.
I've removed the `*args` as well to prevent errors from positional vs keyword argument ordering. the `BusABC` argument `can_filters` could get passed into `Usb2canBus` via `kwargs`.